jobs/config.json

```diff
@@ -869,6 +869,22 @@
       "sig-instrumentation"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-dind-ipv6": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubeadm-dind",
+      "--extract=ci/latest-bazel",
+      "--gcp-zone=us-central1-f",
+      "--ip-mode=ipv6",
+      "--provider=local",
+      "--test_args=--ginkgo.focus=Networking|Services --ginkgo.skip=IPv4|DNS|Networking-Performance|Federation|functioning\\sNodePort|preserve\\ssource\\spod|affinity\\swork|affinity\\sfor --minStartupPods=8",
+      "--timeout=55m"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-network"
```

Comment:
and ownership is managed by putting the prowjob definition in `config/jobs/kubernetes/some-sig/some-yaml.yaml`